src/lib/simulation/birdsNest.ts

```diff
@@ -64,5 +64,4 @@ export const nestTable = new LootTable()
 	.every(birdsNestID)
 	.add(eggNest, 1, 3)
 	.add(ringNests, 1, 32)
-	.add(wysonSeedsNest, 1, 65 / 2)
-	.add(treeSeedsNest, 1, 65 / 2);
+	.add(treeSeedsNest, 1, 65);
```

Comment:
Keep in mind thawt the stuff is 'simulation/' is global, and often used for more than just individual skills/activities.

I would suggest leaving 'nestTable' unchanged,  and making a birdhouseNestTable variable in the `src/lib/skilling/skills/hunter/birdHouseTrapping.ts` file.

Reply:
This was intentional, not just for bird house trapping. The nest table is used in adding skill clues to loot in the current codebase. This adds them to birdhouses, and makes both cases not include wyson boxes. Wyson uses a different implementation to give boxes.